src/interfaces/messages/types.ts

```diff
@@ -0,0 +1,24 @@
+import type { BaseMessageReply } from '../../core/message-reply.js';
+import type { BaseMessage, Descriptor } from '../../core/types.js';
+import type { DwnInterfaceName, DwnMethodName } from '../../core/message.js';
+
+export type MessagesGetDescriptor = Pick<Descriptor, 'interface' | 'method' > & {
```

Comment:
For my understanding: what is the intent of Picking `interface` and `method` when line 6 and 7 specifies it again?

Reply:
removed `Pick` usage